samples/04_gis_analysts_data_scientists/information_extraction_from_cheshire_fire_incident_reports_using_mistral_language_model.ipynb

```diff
@@ -0,0 +1,1208 @@
+{
```

Comment:
<p>The <code>EntityRecognizer</code>&nbsp;model in&nbsp;<code>arcgis.learn</code>&nbsp;can be used with&nbsp;<a href="https://huggingface.co/transformers/v3.0.2/index.html" target="_blank">Hugging Face Transformers</a>&nbsp;or with large language model backbones. For this sample use case, we will use the Mistral model backbone to extract entities from the text.</p><p>Run the command below to see what backbones are supported for the&nbsp;<strong>entity recognition</strong>&nbsp;task.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Esri/arcgis-python-api/pull/2201/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='16'/>